switchbot_ros/package.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<package>
+  <name>switchbot_ros</name>
+  <version>0.0.0</version>
+  <description>use switchbot with ros</description>
+
+  <author email="obinata@jsk.imi.i.u-tokyo.ac.jp">Yoshiki Obinata</author>
+  <maintainer email="k-okada@jsk.t.u-tokyo.ac.jp">Kei Okada</maintainer>
+
+  <license>BSD</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <build_depend>catkin_virtualenv</build_depend>
+  <build_depend>message_generation</build_depend>
+  <build_depend>rospy</build_depend>
+  <build_depend>std_msgs</build_depend>
+  
+  <run_depend>message_runtime</run_depend>
+  <run_depend>rospy</run_depend>
+  <run_depend>std_msgs</run_depend>
```

Comment:
plaase add python-requests (https://github.com/ros/rosdistro/blob/master/rosdep/python.yaml#L4061) and others to run_depends